apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
did you have the time to check why we have a different behavior with backspace or with tapping ?